docs/install.md

```diff
@@ -314,6 +313,76 @@ Select the following tabs for the installation description of your target.
+    - As **Machine** you can select `q35`, but the older default will work as well. We recommend the para-virtualised **VirtIO SCSI** controller, which should be the default. Click **Next**.
+    - On the **Disks** tab, delete the default `scsi0` disk with the red trash bin button, then click **Next**.
+    - Adjust CPU details as required, we recommend to use the default `kvm64` type. Then click **Next**.
+    - While DietPi runs with less, depending on the software you want to install and run within the VM, we recommend at least 1024 MiB memory size, 2048 MiB allows the DietPi system to setup itself without a swap file by default. When done, click **Next**.
+        ??? info "Dynamic memory allocation via ballooning device"
+            The [ballooning device](https://wikipedia.org/wiki/Memory_ballooning) allows Proxmox to dynamically allocate memory from the host system based on actual memory usage within the VM. I.e. you are able to run VMs a higher overall memory size than the host system has (along other host system processes), as long as the VMs do not fully use their full memory at the same time.
+    - Using a network bridge allows the VM to show up as dedicated system in your LAN, which simplifies SSH and network application access. We recommend to use the default para-virtualised **VirtIO** adapter model. When done, click **Next**.
+    - Confirm the VM creation by clicking **Finish**.
+
+    <font size="+2">2. Download, extract and import the DietPi image</font>
+
+    !!! info "Transferring a disk image to Proxmox"
+        A DietPi disk image can be transferred to the Proxmox server via e.g. USB flash drive or by uploading it as CD/DVD ISO image. Since the import needs to be done via console (accessible via web interface and SSH), we guide you through the path of downloading it directly on the Proxmox server.
+
+    - Select the Proxmox node, then click the **Shell** button at the top right corner. Alternatively connect via SSH to the Proxmox server, using the same login credentials you used for the Proxmox web interface.
+    - In the console window, enter the following commands to download the DietPi image, extract it via `p7zip`, import it as disk to your new VM (using the **VM ID** you chose during creation) and make it the boot drive.  
+        _If not done yet, we recommend to upgrade all APT packages to the latest version._
+
+        ```sh
+        apt update
+        apt full-upgrade
+        apt install p7zip
+        curl -O https://dietpi.com/downloads/images/DietPi_Proxmox-x86_64-Bullseye.7z
+        7zr x DietPi_Proxmox-x86_64-Bullseye.7z
+        # Verifying the SHA256 hash
+        sha256sum -c < <(mawk '/SHA256/{print $2"  DietPi_Proxmox-x86_64-Bullseye.qcow2"}' hash.txt)
+        # Import the disk image
+        ID=100 # replace "100" with your VM ID
+        qm importdisk "$ID" DietPi_Proxmox-x86_64-Bullseye.qcow2 local-lvm
```

Comment:
The ID given here should match to the ID given in a screenshot above (see remark to line 344).

Reply:
As said above, the ID is only shown on one of 7 tabs during VM creation. Not sure whether it is reasonable to show only the first of these tabs to refer to it, or showing more than 7 (for the other steps) screenshots, hardly readable or only showing a crop of the browser window?